layers/org/packages.el

```diff
@@ -135,6 +136,24 @@
           "a" 'org-edit-src-abort
           "k" 'org-edit-src-abort))
 
+      (defun spacemacs/org-html-export-annotated ()
+        "Add the Spacemacs specific annotation and export to html."
+        (interactive)
+        (save-excursion
+          (let ((tmpbf (get-buffer-create "*export-tmp*")))
+            (progn
+              (copy-to-buffer tmpbf (point-min) (point-max))
+              (with-current-buffer tmpbf
+                (progn
+                  (when (string= (buffer-substring (point-min) 3) "* ")
+                    (delete-region (point-min) 3))
+                  (goto-char (point-min))
+                  (insert "#+TITLE: ")
+                  (beginning-of-line 2)
+                  (insert spacemacs--org-export-css-annotation)
+                  (org-html-export-to-html)))
+              (kill-buffer tmpbf)))))
```

Comment:
I believe `org-publish` needs to have access to the filename, therefore we can't use temporary buffers. Otherwise how would it know to export `layers/auto-completion/README.org` to `doc/layers/auto-completion/README.html`?